dashboards/ecs/ecs_cluster_dashboard.sp

```diff
@@ -114,7 +114,7 @@ query "ecs_cluster_count" {
 query "ecs_cluster_active_service_count" {
   sql = <<-EOQ
     select
-      count(*) as "Cluster Active Services"
```

Comment:
This is the only real change. Note that I'm changing `count` to `sum`, and then looking at a specific column (`active_services_count`) to get the actual count of active services (as opposed to simply duplicating the `count(*)` from the other query)